src/main/scala/cromwell/util/EnhancedExponentialBackoff.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.util
+
+import com.google.api.client.util.ExponentialBackOff
+
+class EnhancedExponentialBackoff(enhancedBuilder: EnhancedExponentialBackoff.Builder) extends ExponentialBackOff(enhancedBuilder) {
```

Comment:
So normally when you see `Enhanced` it's an implicit class adding functionality (i.e. the polite way of saying `Pimped`). I wouldn't call it this.